.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,52 @@
+name: Bug report
+description: File a bug report
+title: '[Bug]: '
+labels: [bug]
+assignees: ''
+body:
+  - type: input
```

Comment:
```suggestion
  - type: markdown
    attributes:
      value: |
        **To report a Devopness security vulnerability, please [send an email to `social@devopness.com`](mailto:social@devopness.com) with all the details.**

        :warning: Do **NOT** create a public issue on GitHub for security vulnerabilities. See our [security policy]( <LINK TO BE UPDATED SOON>) for more details. :warning:
  - type: input
```

Reply:
I've added this new field explaining that security vulnerabilities should be reported directly to e-mail as suggested.